tools/check_cfg_backup.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+if ls dev/mongooseim_*/etc/ejabberd.cfg.bak 1> /dev/null 2>&1; then
+    echo "Unrestored config exists. Your actions:"
```

Comment:
For me it sounds confusing, as if I were to choose one of the actions. I'd suggest rephrasing it to sthg like: "You should now do the following:", or just simply: "Stop nodes and rebuild dev nodes".